### PR TITLE
DEVDOCS-2548: Add Email Templates Overview

### PR DESCRIPTION
**Pull request by aglensmith**
# [DEVDOCS-2548](https://jira.bigcommerce.com/browse/DEVDOCS-2548)

## What changed?
* Added Email Templates Overview

### CODE REVIEW

**Comment by uwikaiddi** [comment]
@aglensmith, request peer review

File: docs/api-docs/management-apis/email-templates/overview.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+- [Overrides](#overrides)
+- [Resources](#resources)
+
+</div>
+
+This article provides and overview of BigCommerce's transactional email templates and API.
+
+## Editing
+
+To edit transactional email templates in a store's control panel, [log in](https://support.bigcommerce.com/s/article/Logging-Into-Your-Store) and navigate to **Marketing** › [**Transactional Emails**](http://login.bigcommerce.com/deep-links/manage/transactional-emails).
```
Comment (markcmurphy) [inline docs/api-docs/management-apis/email-templates/overview.md:18]: That link gave me a 404, but the link below pulled up the Email Templates menu for me.

```suggestion
To edit transactional email templates in a store's control panel, [log in](https://support.bigcommerce.com/s/article/Logging-Into-Your-Store) and navigate to **Marketing** › [**Email Templates**](http://login.bigcommerce.com/deep-links/manage/marketing/email-templates).

```

File: docs/api-docs/management-apis/email-templates/overview.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    <td style="padding:5px; font-size:12px; border-bottom:solid 1px #CACACA" width="100" align="center">{{ quantity }}</td>
+</tr>
+{{/each}}
+```
+
+[Learn how to edit, test, and preview transactional email templates in the control panel](https://support.bigcommerce.com/s/article/Customizing-Emails).
+
+## API
+
+To manage transactional email templates programmatically, use the [Email Templates API](https://developer.bigcommerce.com/api-reference/store-management/email-templates). For example, to [get a list of email templates](https://developer.bigcommerce.com/api-reference/store-management/email-templates/email-templates/getemailtemplates), send a `GET` request to `/v3/marketing/email-templates`.
```
Comment (markcmurphy) [inline docs/api-docs/management-apis/email-templates/overview.md:40]: `/v3/marketing/email-templates` is returning a 404 in both Postman and Request Runner, as opposed to the "The route is not found, check the URL" message I get when the endpoint doesn't exist. Maybe it's just temporary?

File: docs/api-docs/management-apis/email-templates/overview.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+      "body": "<!DOCTYPE html> <html lang=\"en\"> <head>     <meta charset=\"UTF-8\">     <title>Title</title> </head> <body> <p>     {{lang \"reset_password\" name=store.name}} </p> <br/> <br/> <a href='{{account.reset_password_link}}'>     {{account.reset_password_link}} </a>  </body> </html>",
+      "translations": [...],
+      "subject": "Reset your password at {{store.name}}"
+    }
+  ],
+  "meta": {...}
+}
+```
+
+To [update an email template](https://developer.bigcommerce.com/api-reference/store-management/email-templates/email-templates/updateemailtemplate), send a `PUT` request to `/v3/marketing/email-templates/{template-name}`.
```
Comment (markcmurphy) [inline docs/api-docs/management-apis/email-templates/overview.md:67]: Tried this one, also a 404

**Review by markcmurphy — CHANGES_REQUESTED** [review]

File: docs/api-docs/management-apis/email-templates/overview.md (hunk 1 above)
Comment (aglensmith) [reply docs/api-docs/management-apis/email-templates/overview.md:19]: Its correct. You need to have the experiment turned on.

File: docs/api-docs/management-apis/email-templates/overview.md (hunk 2 above)
Comment (aglensmith) [reply docs/api-docs/management-apis/email-templates/overview.md:41]: It is correct. Its either not rolled out to your store or the experiment is turned on yet.

File: docs/api-docs/management-apis/email-templates/overview.md (hunk 3 above)
Comment (aglensmith) [reply docs/api-docs/management-apis/email-templates/overview.md:68]: Should work now.

**Review by markcmurphy — APPROVED** [review]

File: docs/api-docs/management-apis/email-templates/overview.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+</div>
+
+This article provides an overview of BigCommerce's transactional email templates and API.
+
+
+## Editing
+
+To edit transactional email templates in a store's control panel, [log in](https://support.bigcommerce.com/s/article/Logging-Into-Your-Store) and navigate to **Marketing** › [**Transactional Emails**](http://login.bigcommerce.com/deep-links/manage/transactional-emails).
+
+Use [handlebars expressions](https://developer.bigcommerce.com/api-docs/store-management/email-templates/handlebars) to localize email templates and access dynamic data (like a customer's first name in an order email).
```
Comment (uwikaiddi) [inline docs/api-docs/management-apis/email-templates/overview.md:21]:
```suggestion
Use [Handlebars expressions](https://developer.bigcommerce.com/api-docs/store-management/email-templates/handlebars) to localize email templates and access dynamic data (like a customer's first name in an order email).

```

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**